docs/pages/deploy-a-cluster/helm-deployments/custom.mdx

```diff
@@ -153,25 +157,31 @@ our `public_addr`s are set to `custom.example.com`.
   It will help if you have access to the DNS provider which hosts `example.com` so you can add a `custom.example.com` record
   and point it to the external IP or hostname of the Kubernetes load balancer.
 
-  Don't worry if you can't - you'll just have to remember to replace `custom.example.com` with the external IP or hostname of the Kubernetes load balancer to be able to access Teleport from your local machine.
+  Don't worry if you can't - you'll just have to remember to set the proxy's `public_addr` to the external IP or
+  hostname of the Kubernetes load balancer to be able to access Teleport from your local machine.
```

Comment:
I don't like this section, I'll remove it entirely.

From what I understand the remark means you can set `public_addr` to the load balancer IP address instead of a DNS entry. On such a setup, web browsers won't be able to validate the web UI and TLS multiplexing is not possible without `--insecure` all over the place.